### PR TITLE
TrustAnchorsProvider using KeyStore object directly

### PR DESCRIPTION
**Pull request by tnorimat**

### CODE REVIEW

**Comment by tnorimat** [comment]
Looking at https://github.com/webauthn4j/webauthn4j/pull/165 and the build logs, It seems that the failure of builds does not relevant to this pull-request.

File: webauthn4j-core/src/main/java/com/webauthn4j/anchor/KeyStoreTrustAnchorsProvider.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+/**
+ * Loads {@link AAGUID} key {@link TrustAnchor} {@link Set} value {@link Map} from Java KeyStore object.
+ */
+public class KeyStoreTrustAnchorsProvider extends CachingTrustAnchorsProviderBase {
+
+    // ~ Instance fields
+    // ================================================================================================
+
+    private KeyStore keyStoreObject;
```
Comment (ynojima) [inline webauthn4j-core/src/main/java/com/webauthn4j/anchor/KeyStoreTrustAnchorsProvider.java]: I prefer `keyStore` than `keyStoreObject` for variable name.

Comment (tnorimat) [reply webauthn4j-core/src/main/java/com/webauthn4j/anchor/KeyStoreTrustAnchorsProvider.java]: Yes, I'll rename it as you said.

File: webauthn4j-core/src/main/java/com/webauthn4j/anchor/KeyStoreTrustAnchorsProvider.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import java.util.Map;
+import java.util.Set;
+
+import com.webauthn4j.data.attestation.authenticator.AAGUID;
+import com.webauthn4j.util.AssertUtil;
+
+/**
+ * Loads {@link AAGUID} key {@link TrustAnchor} {@link Set} value {@link Map} from Java KeyStore object.
+ */
+public class KeyStoreTrustAnchorsProvider extends CachingTrustAnchorsProviderBase {
```
[the remark this answers is not in the document]
Comment (tnorimat) [reply webauthn4j-core/src/main/java/com/webauthn4j/anchor/KeyStoreTrustAnchorsProvider.java]: Yes, I'll rewrite this class by implementing `TrustAnchorsProvider` directly.

**Pull request merged**